src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenDeconstructTests.cs

```diff
@@ -227,9 +248,9 @@ static void Main()
 ";
             var comp = CreateCompilationWithMscorlib(source, parseOptions: TestOptions.Regular.WithTuplesFeature());
             comp.VerifyDiagnostics(
-                // (8,18): error CS8208: The Deconstruct method for type 'C.Deconstruct(out int, int)' must have only out parameters.
+                // (8,9): error CS1615: Argument 2 may not be passed with the 'out' keyword
```

Comment:
+1 to simply reporting the appropriate `Deconstruct` method was not found, here and in the `ref` case below. Perhaps the error should report what signature is expected: `No matching 'Deconstruct' method found. Type 'MyClass' must have an accessible 'Deconstruct' instance method with the expected number of 'out' parameters.`